e2etests/boot_tests.bzl

```diff
@@ -0,0 +1,9 @@
+def launch_cvd_boot_test(name, branch, target):
+  native.sh_test(
+    name = name,
+    size = "medium",
+    srcs = ["launch_cvd_boot_test.sh"],
+    args = ["-b", branch, "-t", target],
+    tags = ["exclusive"],
```

Comment:
Might want [`"external"` and `"no-sandbox"`](https://bazel.build/reference/be/common-definitions#common.tags) as well.

Reply:
Done